### DISCUSSION TITLE
Renew ip address every hour

### DISCUSSION THREAD

**Discussion by terrapi23** [qdm12/gluetun#928] [Q&A]
Hi, I am using gluetun in a docker container running jdownloader. My VPN provider is Windscribe and I use the Wireguard protocol. Now I have set gluetun to get a new public IP every hour, but I keep getting the same IP assigned. Is there any way to set it to always be a different IP?

Otherwise compliments for the great work

**Comment by qdm12** [comment]
Have a bunch of filters (for example multiple countries, or no filter at all), and call the HTTP API to restart the vpn every hour.

That will pick a vpn server at random every hour. It won't rotate exactly though.

**Reply by seth100** [reply]
I'm using Mullvad with Wireguard protocol and would like to accomplish that. What API do I have to call to restart the VPN?
